Usage.md

```diff
@@ -214,12 +232,90 @@ litmusctl get agents --project-id=""
 **Output:**
 
 ```
-AGENTID                                AGENTNAME          STATUS 
-55ecc7f2-2754-43aa-8e12-6903e4c6183a   agent-1            ACTIVE 
-13dsf3d1-5324-54af-4g23-5331g5v2364f   agent-2            INACTIVE
+AGENTID                                AGENTNAME          STATUS     REGISTRATION
+55ecc7f2-2754-43aa-8e12-6903e4c6183a   agent-1            ACTIVE     REGISTERED
+13dsf3d1-5324-54af-4g23-5331g5v2364f   agent-2            INACTIVE   NOT REGISTERED
+```
+
+
+* To disconnect an agent, issue the following command..
+```shell
+litmusctl disconnect agent f9799723-29f1-454c-b830-ae8ba7ee4c30 --project-id=""
+```
+
+**Output:**
+
+```
+🚀 ChaosAgent successfully disconnected.
+```
+
+
+* To list the created workflows within a project, issue the following command.
+```shell
+litmusctl get workflows --project-id=""
+```
+
+**Output:**
+
+```
+WORKFLOW ID                          WORKFLOW NAME                    WORKFLOW TYPE     NEXT SCHEDULE AGENT ID                             AGENT NAME LAST UPDATED BY
+9433b48c-4ab7-4544-8dab-4a7237619e09 custom-chaos-workflow-1627980541 Non Cron Workflow None          f9799723-29f1-454c-b830-ae8ba7ee4c30 Self-Agent admin
+
+Showing 1 of 1 workflows
+```
+
+
+* To list all the chaos workflow runs within a project, issue the following command.
+```shell
+litmusctl get workflowruns --project-id=""
+```
+
+**Output:**
+
+```
+WORKFLOW RUN ID                      STATUS  RESILIENCY SCORE WORKFLOW ID                          WORKFLOW NAME                    TARGET AGENT LAST RUN                 EXECUTED BY
+8ceb712c-1ed4-40e6-adc4-01f78d281506 Running 0.00             9433b48c-4ab7-4544-8dab-4a7237619e09 custom-chaos-workflow-1627980541 Self-Agent   June 1 2022, 10:28:02 pm admin
+
+Showing 1 of 1 workflow runs
+```
+
+
+* To describe a particular chaos workflow, issue the following command.
+```shell
+litmusctl describe workflow 9433b48c-4ab7-4544-8dab-4a7237619e09 --project-id=""
```

Comment:
can we also add command without actual workflowID, so that user knows what is - `9433b48c-4ab7-4544-8dab-4a7237619e09`